packages/lib/src/components/BaseElement.ts

```diff
@@ -106,11 +107,13 @@ class BaseElement<P extends BaseElementProps> {
         this.props = this.formatProps({ ...this.props, ...props });
         this.state = {};
 
-        return this.unmount().remount();
+        return this.unmount().mount(this._node); // for new mount fny
     }
 
     /**
-     * Unmounts an element and mounts it again on the same node
+     * Unmounts an element and mounts it again on the same node i.e. allows mount w/o having to pass a node.
+     * Should be "private" & undocumented (although being a public function is useful for testing).
+     * Left in for legacy reasons
```

Comment:
@sponglord can you recall me for which scenario is it useful for testing? Does it make sense to add to our SDK v6 plans to completely remove this method?

Reply:
remount is useful for testing (according to Marc).
And yes - it should be removed for v6